utilities/cheyenne/launch-dask.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os, sys, argparse, subprocess, re, time, logging, socket
+from dask.distributed import Client
+logger = logging.getLogger(__name__)
+
+def parse_command_line(args, description):
```

Comment:
The `setup_jlab` script uses `click` for the parsing the command line arguments. I think you'll find that is an easier/less verbose way to build command line tools.

Reply:
I prefer argparse as it is standard python and I am familiar with it.

Reply:
okay, that is fine. Just so you know, the dask command line tools (`dask-mpi`) use click so that is a dependency this tool already has.

Reply:
In general I think that conventions like this should generally be decided by whoever plans to maintain the code long term.  It's not clear who that is in this situation.  Are you planning to manage bug reports and feature requests on this code long term @jedwards4b ?  Or is future development likely to be left to the Dask/XArray developers?  If the latter then I think that we should stick with `click`.

Reply:
(I'm not saying -1 here though.  I'm not heavily involved in this at the moment.  Happy to go with @jhamman 's decision)